src/Auth/AuthInheritance.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: elena
+ * Date: 28.10.16
+ * Time: 20:27.
+ */
+
+namespace Veta\Homework\Auth;
+
+class AuthInheritance extends Auth
+{
+    /**
+     * @param string $username
+     * @param string $password
+     */
+    public function __construct($username, $password)
```

Comment:
There is no need to create __construct function if you call the parent's method. If this function would not exist - the parent's constructor would be inherited and used.